# 1. 常见的异常
a = [1,2,3]
i = 0
while i < 5:
    print(a[i])
    i = i + 1

print("代码执行到这儿了")
# 2. 断言错误
assert 1 == 1
